iceoryx_dds/include/iceoryx_dds/dds/dds_config.hpp

```diff
@@ -21,6 +21,7 @@ namespace iox
 {
 namespace dds
 {
+using namespace units::duration_literals;
```

Comment:
do we want this in headers?

Reply:
Nope, already this discuss with @elfenpiff we want to avoid namespace pollution and keep ADL time low. `inline namespace` does not prevent this.
Steps in follow-up:
1.  Create c'tor for all numeric literal operators e.g. `Duration::Seconds()`
1. Remove `using namespace` from headers